### ISSUE TITLE
Error: undefined method `normalize_architecture' for Mixlib::Install::Util:Class

### ISSUE THREAD

**Issue by predominant** [chef-cookbooks/chef-ingredient#151]
### Cookbook version
2.0.3

### Chef-client version
12.17.44

### Platform Details
Ubuntu 14.04

### Scenario:
Converge

### Steps to Reproduce:
Converge on any client fails with 2.0.3. Stacktrace below. push-jobs specifically, which depends on chef-ingredient.

### Expected Result:
Successful converge

### Actual Result:

```
================================================================================
    Error executing action `install` on resource 'chef_ingredient[push-jobs-client]'
    ================================================================================
    
    NoMethodError
    -------------
    undefined method `normalize_architecture' for Mixlib::Install::Util:Class
    
    Cookbook Trace:
    ---------------
    /var/chef/cache/cookbooks/chef-ingredient/libraries/helpers.rb:287:in `installer'
    /var/chef/cache/cookbooks/chef-ingredient/libraries/default_handler.rb:96:in `configure_from_channel'
    /var/chef/cache/cookbooks/chef-ingredient/libraries/default_handler.rb:46:in `configure_package'
    /var/chef/cache/cookbooks/chef-ingredient/libraries/default_handler.rb:23:in `handle_install'
    /var/chef/cache/cookbooks/chef-ingredient/resources/chef_ingredient.rb:59:in `block in class_from_file'
    
    Resource Declaration:
    ---------------------
    # In /var/chef/cache/cookbooks/push-jobs/recipes/package.rb
    
     51: chef_ingredient 'push-jobs-client' do
     52:   version package_version || node['push_jobs']['package_version']
     53:   package_source local_package_path if defined?(local_package_path)
     54:   platform_version_compatibility_mode true
     55: end
    
    Compiled Resource:
    ------------------
    # Declared in /var/chef/cache/cookbooks/push-jobs/recipes/package.rb:51:in `from_file'
    
    chef_ingredient("push-jobs-client") do
      action [:install]
      retries 0
      retry_delay 2
      default_guard_interpreter :default
      declared_type :chef_ingredient
      cookbook_name "push-jobs"
      recipe_name "package"
      platform_version_compatibility_mode true
      product_name "push-jobs-client"
    end
    
    Platform:
    ---------
    x86_64-linux
```

**Comment by wrightp** [comment]
mixlib-install gem needs to be set to 3.1 https://github.com/chef-cookbooks/chef-ingredient/blob/master/libraries/helpers.rb#L90

**Comment by wrightp** [comment]
chef-ingredient 2.0.4 published to supermarket with the mixlib-install 3.2 version pin.
